src/features/callAssignments/models/CallAssignmentModel.ts

```diff
@@ -132,6 +132,18 @@ export default class CallAssignmentModel extends ModelBase {
     this._repo.removeCaller(this._orgId, this._id, callerId);
   }
 
+  setCallerAccess(expose_target_details: boolean) {
+    this._repo.updateCallAssignment(this._orgId, this._id, {
+      expose_target_details,
+    });
+  }
+
+  setCallerNotes(disable_caller_notes: boolean) {
+    this._repo.updateCallAssignment(this._orgId, this._id, {
+      disable_caller_notes,
```

Comment:
This needs to be inverted, right? I think the argument should be called `enabled`, and this line should read `disable_caller_notes: !enabled`. Do you agree?

Reply:
oooooh ofcourse it does!